curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/boo-who.portuguese.md

```diff
@@ -8,7 +8,7 @@ localeTitle: Boo quem
 ---
 
 ## Description
-<section id="description"> Verifique se um valor é classificado como um primitivo booleano. Retornar verdadeiro ou falso. Primitivos booleanos são os valores verdadeiro e falso. Lembre-se de usar <a href="http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514" target="_blank">Read-Search-Ask</a> se você ficar preso. Tente programação em par. Escreva seu próprio código. </section>
+<section id="description"> Verifique se um valor é classificado como um primitivo booleano. Retornando verdadeiro ou falso. Primitivos booleanos são os valores verdadeiro e falso. Lembre-se de usar <a href="http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514" target="_blank">Read-Search-Ask</a> se você ficar preso em dúvida. Tente programação em par(duplas). Escreva seu próprio código. </section>
```

Comment:
I think it would sound more natural using these changes:

1. `Verifique se um valor é classificado como um primitivo booleano retornando verdadeiro ou falso.`
2. `... se você tiver dúvidas`
3. `Tente programação em pares (duplas).`